doc/doxygen/src/roadmap.md

```diff
@@ -34,9 +34,49 @@ The text and items below are tentative, up for discussion, to be updated by regu
 # Peripheral drivers
 (contact/steering: [Hauke](https://github.com/haukepetersen))
 
+## Timers
+
+(contact/steering: [kaspar030](https://github.com/kaspar030), [benpicco](https://github.com/benpicco), [maribu](https://github.com/maribu))
+
 - cleanup and unification of low-level timer interfaces (`timer`, `rtt`, `rtc`)
+- implement capture mode
+- add an API to query which frequencies which timer supports
+    - see https://github.com/RIOT-OS/RIOT/pull/16349 for one proposal
+- add an API to query the actual frequency a timer is running at
+    - see https://github.com/RIOT-OS/RIOT/pull/20131 for one proposal
```

Comment:
remove this? 
 since you closed it and the in favor Pr #16349  got merged?

Reply:
Indeed :tada: